### PR TITLE
Add functionality for exclude *

### PR DESCRIPTION
**Pull request by ryanwilliamquinn**
Addresses #23.

### CODE REVIEW

File: lib/squeeze.js [hunk 1]
```diff
@@ -33,8 +33,13 @@ class Squeeze extends Stream.Transform {
             const tags = {};
 
             if (filter && (filter.include || filter.exclude)) {
-                tags.include = Squeeze.toTagArray(filter.include);
-                tags.exclude = Squeeze.toTagArray(filter.exclude);
+                if (filter.exclude === '*') {
```
Comment (arb) [inline lib/squeeze.js]: Why do we need this extra check?

Comment (ryanwilliamquinn) [inline lib/squeeze.js]: If we don't add a check here, the calls to `Squeeze.toTagArray` will return empty arrays for both include and exclude. If both include and exclude are empty arrays, all events are included. I think we will have to add some logic somewhere to handle this case, if we want to include it. We could put it into `toTagArray`, if you prefer.

Comment (ryanwilliamquinn) [inline lib/squeeze.js]: Alternatively, we could check if `*` is included in the either the include or exclude tag, and proceed as expected (exclude overriding if both contain `*`). That might make more sense, that way the tag array is always an array, as the name implies.

File: test/index.js [hunk 2]
```diff
@@ -140,6 +140,27 @@ describe('Squeeze', () => {
             done();
         });
 
+        it('returns false if this reporter should not report this event type (exclude "*")', { plan: 1 }, (done) => {
+
+            const subscription = Squeeze.subscription({ request: { exclude: '*' } });
```
Comment (benleen) [inline test/index.js]: Wouldn't it be better if just did not allow `exclude: '*'`? The behaviour you expect here is the default behaviour after all.

**Comment by benleen** [comment]
I don't see a use case for using `{ exclude: '*' }`. Could you maybe give some examples?

**Comment by benleen** [comment]
Just went into the use case of the issue you opened (I should've done that to begin with, sorry).

In light of that usage it could make sense, however, at the same time I would discourage users from using that kind of configuration because it's nonsensical in most cases.

**Comment by hueniverse** [comment]
Still being considered?

**Pull request closed without merging**